eslint-precommit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "extends": [
+    "../mastarm/lib/eslintrc.json"
+  ],
+  "parser": "babel-eslint",
+  "plugins": [
+    "sort-destructure-keys"
+  ], 
+  "rules": {
+    "react/jsx-sort-props": [
+      "error",
+      {
+        "ignoreCase": true
+      }
+    ],
+    "sort-destructure-keys/sort-destructure-keys": [
+      "error",
+      {
+        "caseSensitive": false
+      }
+    ],
+    "sort-keys": [
+      "error",
+      "asc",
+      {
+        "caseSensitive": false
+      }
+    ],
+    "sort-vars": [
+      "error",
+      {
+        "ignoreCase": true
+      }
+    ]
+  }
```

Comment:
I didn't realize or forgot about the existence of https://github.com/conveyal/mastarm/pull/308. We shouldn't be added extra linting rules in here if they can be part of the overall linting rules. Furthermore, our tests won't pick up these linting rules. I'll try to contribute to https://github.com/conveyal/mastarm/pull/308 so that a new release of mastarm can be made that includes these new rules.

Reply:
Great, so the extra rules and this file are no longer needed (https://github.com/opentripplanner/otp-react-redux/pull/377/commits/e66a45da926abaae948d9460fd49eeee52417e70).